runtime/src/utils/CurriedProviders.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export const CurriedProviders = ({ providers, children }) =>
+    Providers.reduceRight(
+        (acc, Provider) => <Provider>{acc}</Provider>,
```

Comment:
The above is incorrect. @amcgee has provided the correct reason why the names are like that:
> Since each provider is actually exported by its respective service package with the name Provider

Will resolve this